client/src/app/anagram/word.service.ts

```diff
@@ -32,7 +32,9 @@ export class WordService {
         httpParams = httpParams.set(this.filterTypeKey, filters.filterType);
       }
       if (filters.length) {
-        httpParams = httpParams.set(this.lengthKey, filters.length);
+        if (filters.length > 0 ) {
```

Comment:
This is where the bug fix is. It should ideally be fixed with setting limits on the form field, but this works as a patch